contentcuration/contentcuration/collectstatic_settings.py

```diff
@@ -0,0 +1,6 @@
+# Settings  used by containers running collectstatic. Scope our services
+# to the only ones needed to run collectstatic.
+
+from .settings import *
+
+CACHES['default']['BACKEND'] = "django_prometheus.cache.backends.locmem.LocMemCache"
```

Comment:
We create a new setting so we can granularly control what services we connect to. In this case, we don't need any production settings.

Reply:
that's convenient!